apps/judicial-system/backend/src/app/modules/event/event.service.ts

```diff
@@ -83,14 +83,14 @@ export class EventService {
         event === CaseEvent.ACCEPT && isIndictmentCase(theCase.type)
           ? caseEvent[CaseEvent.ACCEPT_INDICTMENT]
           : `${caseEvent[event]}${eventOnly ? ' - aðgerð ekki framkvæmd' : ''}`
-      const typeText = `${capitalize(
+      const typeText = `${capitalize(caseTypes[theCase.type])}${
         isIndictmentCase(theCase.type)
-          ? readableIndictmentSubtypes(
+          ? `:(${readableIndictmentSubtypes(
```

Comment:
Extreme nitpick, but consider adding a space after the colon, or moving the colon and space behind the list of indictment subtypes 

So we would see either 
Ákæra: (umferðarlagabrot) db1ac914-a7b2-4b6d-a10d-e099fd2f3bcb
or 
Ákæra (umferðarlagabrot): db1ac914-a7b2-4b6d-a10d-e099fd2f3bcb

Feels easier on the eyes to read 🤓

Reply:
My vote is on the second one 🥈